src/argus_htmx/incidents/customization.py

```diff
@@ -38,8 +46,18 @@ class IncidentField:
 # Fields for demo/debug/dev purposes
 TEMP_FIELDS = {
     "id": IncidentField("id", "ID", "htmx/incidents/_incident_pk.html"),
-    "tag": IncidentField("tag", "Tag", "htmx/incidents/_incident_tag.html", context={"tag": "location"}),
-    "ack": IncidentField("ack", "Ack", "htmx/incidents/_incident_ack.html")
+}
+
+EXTRA_FIELDS = {
+    "location_tag": IncidentField("location", "Location", "htmx/incidents/_incident_tag.html",
+                                  context=FieldContext(tag={"key": "location"})),
+    "problem_type_tag": IncidentField("problem_type", "Problem Type", "htmx/incidents/_incident_tag.html",
+                                      context=FieldContext(tag={"key": "problem_type"})),
+    "any_ack": IncidentField("ack", "Ack", "htmx/incidents/_incident_ack.html"),
+    "sd_ack": IncidentField("sd_ack", "SD Ack", "htmx/incidents/_incident_ack.html",
+                            context=FieldContext(ack="sd")),
+    "noc_ack": IncidentField("noc_ack", "NOC Ack", "htmx/incidents/_incident_ack.html",
+                             context=FieldContext(ack="noc")),
```

Comment:
This is hardcoding both ours and geant's ack-systems. Hardcode ours, as just "ack", show how to do geant's in doc.